### PR TITLE
Upgrade ember-cli to 0.1.12

### PR DESCRIPTION
**Pull request by knownasilya**
Also update to the new ember-cli-qunit api.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -8,10 +8,10 @@
     "ember-resolver": "~0.1.11",
     "loader.js": "ember-cli/loader.js#1.0.1",
     "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "rwjblue/ember-cli-test-loader#0.0.4",
+    "ember-cli-test-loader": "ember-cli/ember-cli-test-loader#0.1.1",
     "ember-load-initializers": "ember-cli/ember-load-initializers#0.0.2",
-    "ember-qunit": "0.1.8",
-    "ember-qunit-notifications": "0.0.4",
-    "qunit": "~1.15.0"
+    "ember-qunit": "0.2.6",
```
Comment (rwjblue) [inline bower.json]: Current is 0.2.8 (just released).

**Comment by rwjblue** [comment]
Can you:

```javascript
import { module, test } from 'qunit';
```

At the beginning of the test file (that just landed in the ember-cli blueprints)?

**Comment by knownasilya** [comment]
@rwjblue is that fine?

File: bower.json (hunk 1 above)
Comment (knownasilya) [inline bower.json]: Done

**Comment by rwjblue** [comment]
Thanks!

**Pull request merged**